routes/candidates.js

```diff
@@ -270,6 +283,24 @@ router.post('/create/:rcs_id/:office_id', function (req, res) {
             logger.write(connection, req.session.cas_user, "CANDIDATE_CREATE", "Added " + cms_data.username +
                 " as a candidate for office #" + office_id);
 
+            let mailoptions = {
+                text: "Hello " + cms_data.first_name + ",\n \n" +
+                "Your profile on the Elections website has been activated.\n" +
+                "You can sign into the Elections website with your RCS ID and edit your page here: https://elections.union.rpi.edu/candidate/" + cms_data.username + " \n" +
+                "If you are running for more than one office, you will get this email every time a new office is added to your profile. \n" +
```

Comment:
Lol. Let's make an issue to track this so we can eventually fix it. Probably wouldn't take much—just a check for this candidate having already been added for any office, and then changing the message to reflect that they've been added as a candidate instead of saying their profile is activated.